LA4Ice.c

```diff
@@ -283,7 +283,7 @@ int main(int argc, char *argv[])
   }
 
   //  Read the file and display selected records
-  
+  if (tspace > 0)
```

Comment:
tspace seems like the number of records in las file.
See line 328, an divide by zero error would raise if input file has no record.

This change just bails out early when input is empty.

The "+ +\n- -\n" is used as stop sign in LA4ice output when -m is specified. In case input is an empty file, we still need to write this stop sign to the output.